bootstrap/cmd/kfctlClient/main.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	log "github.com/golang/glog"
+	"github.com/kubeflow/kubeflow/bootstrap/cmd/bootstrap/app"
+	"github.com/kubeflow/kubeflow/bootstrap/pkg/kfapp/gcp"
+	"github.com/kubeflow/kubeflow/bootstrap/pkg/utils"
+	kfdefsv2 "github.com/kubeflow/kubeflow/bootstrap/v2/pkg/apis/apps/kfdef/v1alpha1"
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+	"os"
+
+	// log "github.com/sirupsen/logrus"
+	"golang.org/x/oauth2/google"
+	dm "google.golang.org/api/deploymentmanager/v2"
+)
+
+func init() {
+	// Add filename as one of the fields of the structured log message
+	//filenameHook := filename.NewHook()
+	//filenameHook.Field = "filename"
+	//log.AddHook(filenameHook)
+}
+
+// ServerOption is the main context object for the controller manager.
+type ServerOption struct {
+	Project  string
+	Name     string
+	Config   string
+	Endpoint string
+	Zone     string
+}
+
+// NewServerOption creates a new CMServer with a default config.
+func NewServerOption() *ServerOption {
+	s := ServerOption{}
+	return &s
+}
+
+// AddFlags adds flags for a specific Server to the specified FlagSet
+func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
+
+	fs.StringVar(&s.Config, "config", "", "Path to a YAML file describing an app to create on startup.")
```

Comment:
is there a config you will point to?  I might be able to use it for load test as well, or at least for my reference.

Reply:
I think we need to use a config file available from release tag.

Reply:
You can specify the URI of what ever KfDef file you want to use.
I set a default value to point to the GCP value on master.